ggshield/cmd/secret/scan/range.py

```diff
@@ -32,7 +32,7 @@ def range_cmd(
 
     Any git compatible commit range can be provided as an input.
 
-    Example: `ggshield secret scan commit-range HEAD~1...`
```

Comment:
This is a valid git range:

```
$ ggshield secret scan commit-range HEAD~1... -v
Commits to scan: 1
Scanned changelog.d/20241129_155452_samuel.guillaume_spi_526_implement_deletion_commit_identification_in_ggshield.md
Scanned ggshield/verticals/secret/secret_scan_collection.py
Scanned ggshield/verticals/secret/secret_scanner.py
Scanned b34d26479d3fd44fb07d53ddd11ae9bf8957524c
Scanning... ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 1 / 1

secrets-engine-version: 2.127.0

No secrets have been found
```

See [git-rev-list doc](https://git-scm.com/docs/git-rev-list#_description) for more details.